ecs/daemon/parsers.conf

```diff
@@ -0,0 +1,6 @@
+[PARSER]
+    Name                syslog
+    Format              regex
+    Regex               ^(?<time>[^ ]* {1,2}[^ ]* [^ ]*) (?<host>[^ ]*) (?<ident>[a-zA-Z0-9_\/\.\-]*)(?:\[(?<pid>[0-9]+)\])?(?:[^\:]*\:)? *(?<message>.*)$
```

Comment:
For documentation here's an example that shows that this works to be used with Firelens datajet or similar
```
May 11 10:40:48 scrooge disk-health-nurse[26783]: [ID 702911 user.error] m:SY-mon-full-500 c:H : partition health measures for /var did not suffice - still using 96% of partition space
```